internal/peer/gossip/mcs.go

```diff
@@ -45,6 +51,8 @@ type MSPMessageCryptoService struct {
 	localSigner                identity.SignerSerializer
 	deserializer               DeserializersManager
 	hasher                     Hasher
+	channelConfigGetter        ChannelConfigGetter
+	verifiers                  sync.Map // channel (string) -> verifier func
```

Comment:
I am not sure it's cost-effective to have the verifiers. 

Since `channelConfigGetter` is a map lookup, `s.channelPolicyManagerGetter.Manager` is also a memory lookup, I don't see the point in maintaining a map of verifiers just for this. Why not always use the getter, construct the policy manager and use it? There is no I/O or heavy computation anyhow.

Reply:
So IMO we can also remove that callback altogether.